src/locales/en/v1/effects-and-envelopes.adoc

```diff
@@ -18,7 +18,7 @@ image::../media/U2/NewEnvelope.png[Alt Text]
 [[envelopeswithseteffect]]
 === Envelopes with setEffect
 
-The `setEffect()` function takes a variable number of arguments. Its full set of arguments is listed below.
+The `setEffect()` function takes a variable number of parameters. Its full set of parameters is listed below.
```

Comment:
@heerman Could we improve this further by substituting the word "variable" for something else? It seems to be mixing in even more potential confusing programming jargon when it is really just trying to say that it has multiple method/function signatures. Hmm... 🤔